src/main.py

```diff
@@ -261,25 +32,25 @@ def gettingDataFromProfiles():
     for linkedin_url in linkedin_urls:
 
         driver.get(linkedin_url)
-        time.sleep(5)
+        WebDriverWait(driver,30).until(EC.presence_of_element_located((By.ID, "experience")))
         src = driver.page_source
         soup = BeautifulSoup(src, 'html.parser')
 
 
         # try:
         names = []
         users_id = []
-        userid = linkedin_url.split('in/')[1].split('/')[0]
-        name = extracting_from_personal_info_section_from_profile(soup)
+        userid = driver.current_url.split('in/')[1].split('/')[0]
+        name = getting_data_from_a_profile.extracting_from_personal_info_section_from_profile(soup)
 
         num_jobs = []
         num_degrees = []
 
 
-        extracting_from_experience_section_from_profile(soup,driver,all_experience_data,person_num,num_jobs)
-        extracting_from_education_section_from_profile(soup,all_education_data,person_num,num_degrees)
+        getting_data_from_a_profile.extracting_from_experience_section_from_profile(soup, driver, all_experience_data, person_num, num_jobs)
+        getting_data_from_a_profile.extracting_from_education_section_from_profile(soup, all_education_data, person_num, num_degrees)
 
-        for i in range(num_jobs):
+        for i in range(num_jobs[0]):
```

Comment:
The name `num_jobs` makes me feel that this is an integer value. 
Also, it is not clear why you have it as an array when you are always only going to be using the first value of the array.